apis/pom.xml

```diff
@@ -32,7 +32,7 @@
       Please maintain default values for int-test in sync with io.stargate.sgv2.docsapi.testresource.StargateTestResource.Defaults
```

Comment:
versions in the APIs are updated in the separate PR, targeting the `main` branch.. That's done after this PR is merged..

Reply:
Reverted the version here to `4.0.4`.